src/hooks/useArticleTextSize.js

```diff
@@ -1,12 +1,18 @@
-import { useState, useLayoutEffect } from 'preact/hooks'
+import { useState, useEffect } from 'preact/hooks'
 import { articleTextSize } from 'utils'
 
-export const useArticleTextSize = (dependencies = []) => {
-  const [textSize, setTextSize] = useState(articleTextSize.get())
+let setArticleFontClass

```

Comment:
Thanks for the details explanation about why this is there. I understand where you are coming from but sadly that won't do ;)

This is a great opportunity to review why and how the preact components render and to put in place the proper components to make the article text behind the text size popup update in real time.

Reply:
Got it. I'm now switching to using the global context (https://github.com/wikimedia/wikipedia-kaios/pull/300/commits/459b49e94437875efd8a5b6d800414b56d8b5ca5)